### ISSUE TITLE
AddError Path seems not working

### ISSUE THREAD

**Issue by syssam** [99designs/gqlgen#596]
**Code**
```
path := graphql.GetResolverContext(ctx).Path()
path = append(path, 1, "FieldName")
graphql.AddError(ctx, &gqlerror.Error{
	Message: "Error 1",
	Path:    path,
})
```

**Expected Behaviour**
```
{
  "data": {
    "todo": null
  },
  "errors": [
    { "message": "Error 1", "path": [ "todo", 1, "FieldName" ] },
  ]
}
```

**Current Behaviour**
```
{
  "data": {
    "todo": null
  },
  "errors": [
    { "message": "Error 1", "path": [ "todo"] },
  ]
}
```

**Comment by mathewbyrne** [comment]
Although this is fixed in #620, it's probably worth noting that this isn't intended to be affected, and the library will automatically set this for you. I'd be interested to know what the use-case is for setting the path in a resolver.

**Comment by jszwedko** [comment]
@mathewbyrne not the original author, but I have a use case where an error might occur serializing one particular field of the struct being returned in a resolver method. In this case I'd like to still return the other fields, with an error for the one that couldn't serialize.
